boards/arm/stm32/stm32f4discovery/configs/sbutton/defconfig

```diff
@@ -0,0 +1,56 @@
+#
```

Comment:
Shouldn't this defconfig define an absolute minimum needed for sbutton to work? Also, was it indeed generated with `make savedefconfig`? I would expect entries like `CONFIG_RAM_START=0x20000000` to be defined in default value so it should not be present here?